piker/brokers/kraken.py

```diff
@@ -801,6 +846,29 @@ async def process_order_msgs(
         # HINT: create a ``pydantic.BaseModel`` to parse and validate
         # and then in the caller recast to our native ``BrokerdX`` msg types.
 
+        # check that we are on openOrders stream XXX: Maybe do something with sequence
```

Comment:
i wouldn't put this here. this is exactly why i added the `process_order_msgs()` stream handler in the previous commit.